2.0/s2i/bin/run

```diff
@@ -10,6 +10,16 @@ set -e
 export RACK_ENV=${RACK_ENV:-"production"}
 
 if is_puma_installed; then
+  export_str=$(cgroup-limits.py)
```

Comment:
The check below is for something else (will not use it). But the fact that it's on a separate line is the difference. `export_str=$(cgroup-limits.py) ; export $export_str` will fail and the script will exit if cgroup-limits.py fails (because of `set -e`). If you try `export $(cgroup-limits.py)` it will ignore cgroup-limits.py failure and will try to work with it's output anyway.